jormungandr-lib/src/interfaces/value.rs

```diff
@@ -0,0 +1,135 @@
+use chain_impl_mockchain::value;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use std::{fmt, str::FromStr};
+
+/// Value in the blockchain, always printed as absolute Lovelace
+///
+/// Value has some property to be human readable on standard display
+///
+/// ```
+/// # use jormungandr_lib::interfaces::Value;
+/// # use chain_impl_mockchain::value::Value as StdValue;
+///
+/// let value: Value = StdValue(64).into();
+///
+/// println!("value: {}", value);
+///
+/// # assert_eq!(value.to_string(), "64");
+/// ```
+///
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct Value(value::Value);
```

Comment:
Same thing with `Value`: what's the added value?

Reply:
`pub use value::Value;` is a one-line way to re-export a type from another crate.

Reply:
again, not the right interfaces/encoding/decoding